xclim/indices/_anuclim.py

```diff
@@ -0,0 +1,599 @@
+    tas: xarray.DataArray,
+    op: str = None,
+    input_freq: str = None,
+    freq="YS",
+):
+    r""" ANUCLIM Total precipitation of warmest/coldest quarter
+    The warmest (or coldest) quarter of the year is determined, and the total
+    precipitation of this period is calculated.  If the input data frequency is "daily" or "weekly" quarters
+    are defined as 13 week periods, otherwise are 3 months.
+
+    Parameters
+    ----------
+    pr : xarray.DataArray
+      Total precipitation rate at daily, weekly, or monthly frequency.
+    tas : xarray.DataArray
+      Mean temperature [℃] or [K] at daily, weekly, or monthly frequency.
+    op : str
+      Operation to perform: 'warmest' calculate for the warmest quarter ; 'coldest' calculate for the coldest quarter.
+    input_freq : str
+      Input data time frequency - One of 'daily', 'weekly' or 'monthly'.
+    freq : str
+      Resampling frequency; Defaults to "YS".
+
+    Returns
+    -------
+    xarray.DataArray
+       Total precipitation values of the warmest/coldest quarter of each year.
+
+    Notes
+    -----
+    According to the ANUCLIM user-guide https://fennerschool.anu.edu.au/files/anuclim61.pdf (ch. 6), input
+    values should be at a weekly (or monthly) frequency.  However, the xclim.indices implementation here will calculate
+    the result with input data with daily frequency as well. As such weekly or monthly input values, if desired,
+    should be calculated prior to calling the function.
+
+    """
+    # determine input data frequency
+    if input_freq == "monthly":
+        wind = 3
+        pr = pint_multiply(pr, 1 * units.month, "mm")
+    elif input_freq == "weekly":
+        pr = pint_multiply(pr, 1 * units.week, "mm")
+        wind = 13
+    elif input_freq == "daily":
+        tas = xci.tg_mean(tas, freq="7D")
+        pr = xci.precip_accumulation(pr, freq="7D")
+        wind = 13
+    else:
+        raise NotImplementedError(
+            f'Unknown input time frequency "{input_freq}" : input_freq parameter must be '
+            f'one of "daily", "weekly" or "monthly"'
+        )
+
+    with xarray.set_options(keep_attrs=True):
+        tas_qrt = tas.rolling(time=wind, center=False).mean()
+        pr_qrt = pr.rolling(time=wind, center=False).sum()
+
+        if op == "warmest":
+            np_op = "max"
+        elif op == "coldest":
+            np_op = "min"
+        else:
+            raise NotImplementedError(
+                f'Unknown operation "{op}" ; op parameter but be one of "warmest" or "coldest"'
+            )
+        out = (
+            xarray.Dataset(data_vars={"tas": tas_qrt, "pr": pr_qrt})
+            .resample(time=freq)
+            .map(_get_from_other_extreme, args=("pr", "tas", np_op))
+        )
+        out.attrs = pr.attrs
+        out.attrs["units"] = "mm"
+        return out
+
+
+@declare_units("mm", pr="[precipitation]")
+def prcptot(pr: xarray.DataArray, input_freq: str = None, freq: str = "YS"):
```

Comment:
So this does the same as `precip_accumulation` but with the `input_freq` flag added? I'm not sure it's a good idea to add  it to the `xclim.indices` namespace...  We could maybe add the `input_freq` management to the existing  precip_accumulation`? Or rename this to remove confusion between the two? Or forbid `input_freq="daily"`?

Reply:
I agree not too sure what to do here really.... I originally wanted to just do a wrapped partial of precip_accum in the anuclim module but the return applies a ```pint_multiply(pr, 1 * units.day, "mm")```  so if input data is weekly for example it ends up dividing by 7  ...   I thought it was weird to add support for other frequencies to a single index of our 'standard' library (i.e. non anuplin indices)..  `

I can remove daily support but would there a way for me to wrap two separate calls into the "P12_AnnualPrecip" function in the mapped module..  right now it does this ``"P12_AnnualPrecip": wrapped_partial(indices.prcptot, freq="YS")``   would need a lambda if else kind of thing??

Reply:
Maybe you could move the input_freq-dependent part of the code to the `__build_anuclim` function, so its available to the Anuclim module but never made public?

Reply:
> Maybe you could move the input_freq-dependent part of the code to the `__build_anuclim` function, so its available to the Anuclim module but never made public?

Euhh, Might need some help there ...  :/   Sounds good but not sure how to implement... 

Would this mean the _anucplin.prcptot() would be removed? (Great if possible) and we could somehow shoehorn precip_accum to do what we want ?